tools/nnyeah/nnyeah/AssemblyComparator/ItemFoundEventArgs.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+
+#nullable enable
+
+namespace Microsoft.MaciOS.AssemblyComparator {
+	public class ItemNotFoundEventArgs<T> {
+		public ItemNotFoundEventArgs (string original)
+		{
+			Original = original;
+		}
+		public string Original { get; init; }
+	}
+
+	public class ItemFoundEventArgs<T> {
```

Comment:
Same.

Reply:
Weird that it compiles and runs without it.